require.js

```diff
@@ -1,26 +1,25 @@
 (function() {
   'use strict';
 
-  var globals = typeof window === 'undefined' ? global : window;
+  var globals = new Function('return this')();
   if (typeof globals.require === 'function') return;
 
   var modules = {};
   var cache = {};
   var aliases = {};
-  var has = ({}).hasOwnProperty;
+  var has = {}.hasOwnProperty;
```

Comment:
Are you 100% sure about that? There were a few problems in some browsers because of that.